featuretools/primitives/standard/rolling_transform_primitive.py

```diff
@@ -62,13 +97,19 @@ def __init__(self, window_length=3, gap=0, min_periods=1):
         self.gap = gap
         self.min_periods = min_periods
 
+    def _offset_max(self, series):
+        return _apply_roll_with_offset_gap(series, self.gap, max, self.min_periods)
+
```

Comment:
I'm not sure it matters too much, but do we need these `_offset*` functions as methods on the primitive? I haven't tested this to confirm, but I think you can just call `_apply_roll_with_offset_gap` directly in the `rolled_series.apply` call below, passing in the additional required arguments. Something like this:

```
if isinstance(self.gap, str):
    additional_args = (self.gap, max, self.min_periods)
    return rolled_series.apply(_apply_roll_with_offset_gap, args=additional_args)
```

Reply:
Good call! Updated to do this